install.sh

```diff
@@ -219,16 +219,14 @@ nvm_detect_profile() {
 
   local DETECTED_PROFILE
   DETECTED_PROFILE=''
-  local SHELLTYPE
-  SHELLTYPE="$(basename "/$SHELL")"
 
-  if [ "$SHELLTYPE" = "bash" ]; then
+  if [ -n "$BASH_VERSION" ]; then
```

Comment:
let's use `"${BASH_VERSION-}"` just in case `set -u` is set.

Reply:
updated